packages/stark-ui/src/modules/dropdown/components/dropdown.component.ts

```diff
@@ -172,13 +222,60 @@ export class StarkDropdownComponent extends AbstractStarkUiComponent implements
 		if (changes["multiSelect"]) {
 			this.isMultiSelectEnabled = StarkComponentUtil.isInputEnabled(this.multiSelect);
 		}
+
+		if (changes["isDisabled"]) {
+			if (typeof this.dropdownFormControl !== "undefined") {
```

Comment:
Maybe you can use the type-guard you use in the `ngOnInit` here instead: `StarkFormUtil.isFormControl(this.dropdownFormControl)` so we really check for a formControl

Reply:
good catch, ty